CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Changelog
 
+## v0.34.11
+
+*June 18, 2021*
+
+This release improves the robustness of statesync; tweaking channel priorities and timeouts and
+adding two new parameters to the state sync config.
+
+Friendly reminder, we have a [bug bounty program](https://hackerone.com/tendermint).
```

Comment:
Do we really need this for every release - Can't we just add it once at the top of the changelog?

Reply:
yea, I agree here. I guess for now leave it.